### PR TITLE
Add new file read-a-book.md

### PR DESCRIPTION
**Pull request by aaa11235**

### CODE REVIEW

File: english/read-a-book/read-a-book.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Or you could just read a book!
```
Comment (casey-collab) [inline english/read-a-book/read-a-book.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by aaa11235** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by aaa11235** [comment]
How about now, @casey-collab?

File: english/love/love.md [hunk 2]
```diff
@@ -13,4 +13,6 @@ out of love, make the best of now while being aware of the worst of before.
 Of course, there is no substitute for the experince of it. It must be
 lived in order to be understood. All the words there are in the world 
 cannot make you understand the feeling of it. Anyways, [here's 
-something comepletely different.](../marshmallow.md)
\ No newline at end of file
+something comepletely different.](../marshmallow.md)
+
+If all else fails, you could [try this.](english/read-a-book/read-a-book.md)
```
Comment (casey-collab) [inline english/love/love.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/read-a-book/read-a-book.md (hunk 1 above)
Comment (casey-collab) [inline english/read-a-book/read-a-book.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by aaa11235** [comment]
Third time's a charm, @casey-collab?

**Comment by casey-collab** [comment]
Looks great! :icecream:

**Pull request merged**